cmd/influx/main.go

```diff
@@ -17,21 +17,27 @@ import (
 	"github.com/spf13/viper"
 )
 
-var influxCmd = &cobra.Command{
```

Comment:
want to make this noted, the init and global state are 100% not needed. Only adds side effects and indirection in this case. Creating constructors allows the ability to provide DI for testing (see [pkg_test.go](https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg_test.go))